test/decl/ext/protocol.swift

```diff
@@ -947,7 +947,7 @@ enum Foo : Int, ReallyRaw {
 protocol BadProto1 { }
 protocol BadProto2 { }
 
-extension BadProto1 : BadProto2 { } // expected-error{{extension of protocol 'BadProto1' cannot have an inheritance clause}}
+extension BadProto1 : BadProto2 { } // expected-warning{{inheritance clause in extension of protocol 'BadProto1'. use -enable-conforming-protocol-extensions to remove this warning}}
```

Comment:
This cannot be a warning. This needs to remain an error that goes away when the flag is enabled.

Reply:
That can easily be changed if this ever nears being merged. I left it was a warning for testing while getting the code for gating the feature in place. Thanks for taking a look!